### PR TITLE
Add 363 release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: docs/src/main/sphinx/release/release-363.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Release 363 (XX Oct 2021)
+
+## General
+
+* Reduce query memory usage when spill occurs. ({issue}`9270`)
+* Reduce join memory usage. ({issue}`9327`)
+* Reduce query memory usage during join unspilling. ({issue}`9275`)
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-363.md:7]: #9270 is about unspilling too.  merge?

File: docs/src/main/sphinx/release/release-363.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix decimal arithmetic to not produce negative zeros. Previously comparing two zero decimal
+  values might have returned `false` because of dual zero representation. ({issue}`8985`)
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-363.md:10]: decimal "negative zeros" is not something end user should be able to understand. From user perspective, such thing has never existed.

File: docs/src/main/sphinx/release/release-363.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+* Fix column type reported in `cursor.description` for `time with time zone` column. ({issue}`9460`)
+
+## BigQuery connector
+
+* Fix failure for queries where predicate on `geography`, `array` or `struct` column is pushed down to BigQuery. ({issue}`9391`)
+
+## Cassandra connector
+
+* Add support for Cassandra TUPLE type. ({issue}`8570`)
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-363.md:24]: `tuple`

File: docs/src/main/sphinx/release/release-363.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Elasticsearch connector
+
+* Add support for `scaled_float` type. ({issue}`9358`)
+
+## Hive connector
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix insert overwrite operation on S3 backing storage. ({issue}`9234`)
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-363.md:33]: Is it "Fix" or "Add support for"? @aczajkowski ?

File: docs/src/main/sphinx/release/release-363.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+* Fix `Failed reading parquet data: Socket is closed by peer` query failure when reading from Parquet table with a predicate. ({issue}`9097`)
+* Fix query failure when updating or deleting from ORC ACID transactional table that has some rows deleted since last major compaction. ({issue}`9354`)
+* Fix failure when reading large Parquet files. ({issue}`9469`)
+
+## Iceberg connector
+
+* Add support for comment when adding a new column. ({issue}`9123`)
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-363.md:40]: This feels like "Fix", since previously the comment was silently ignored.

**Review by findepi — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

File: docs/src/main/sphinx/release/release-363.md (hunk 1 above)
Comment (sopel39) [reply docs/src/main/sphinx/release/release-363.md:7]: yup. should be both https://github.com/trinodb/trino/pull/9275

File: docs/src/main/sphinx/release/release-363.md (hunk 2 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-363.md:10]: How about:

> Fix incorrect result for comparisons between zero-valued decimals.

File: docs/src/main/sphinx/release/release-363.md (hunk 4 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-363.md:33]: Also, if it's "fix", what was the issue? Query failure? Data loss? Etc.

File: docs/src/main/sphinx/release/release-363.md (hunk 5 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-363.md:40]: How about:

> Fix support for comments when adding a new column. Previously, they were silently ignored.

File: docs/src/main/sphinx/release/release-363.md (hunk 4 above)
Comment (aczajkowski) [reply docs/src/main/sphinx/release/release-363.md:33]: Yes it's fix. There was no inf. in docs that we don't support overwrite in S3 compatible storages. And it was explicitly denied by check in code. 
It looks that check was added to prevent wrong behaviour in such cases. This PR fixed behaviour and removed check in code denying this operation.

Comment (aczajkowski) [reply docs/src/main/sphinx/release/release-363.md:38]: And behaviour before fix was not removing old data. And effectively we got old and new records in table.

File: docs/src/main/sphinx/release/release-363.md [hunk 6]
```diff
@@ -21,24 +24,27 @@
...
-* Add support for Cassandra TUPLE type. ({issue}`8570`)
+* Add support for Cassandra tuple type. ({issue}`8570`)
 
 ## Elasticsearch connector
 
 * Add support for `scaled_float` type. ({issue}`9358`)
 
 ## Hive connector
 
+* Allow using `hive.insert-existing-partitions-behavior` set to `OVERWRITE` when inserting into tables which
```
Comment (martint) [inline docs/src/main/sphinx/release/release-363.md:35]: This needs better phrasing. @mosabua, any suggestions?

**Comment by martint** [comment]
Added a few more items and adjusted some based on comments. Please take another look.

File: docs/src/main/sphinx/release/release-363.md (hunk 4 above)
Comment (aczajkowski) [reply docs/src/main/sphinx/release/release-363.md:38]: This actually looks redundant with this one: https://github.com/trinodb/trino/pull/9495#discussion_r722453351

File: docs/src/main/sphinx/release/release-363.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+# Release 363 (5 Oct 2021)
+
+## General
+
+* Add `http` event listener implementation which sends JSON serialized events to a remote HTTP endpoint. ({issue}`8821`)
+* Improve performance of queries that group by `bigint` columns. ({issue}`9510`)
+* Reduce query memory usage when spill occurs. ({issue}`9270`, {issue}`9275`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:7]: spilling occurs ?

File: docs/src/main/sphinx/release/release-363.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+* Reduce join memory usage. ({issue}`9327`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:8]: That sounds weird ..  isnt it more correct to say

Reduce memory usage for processing JOIN clauses

or something similar

File: docs/src/main/sphinx/release/release-363.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix incorrect result for comparisons between zero-valued decimals. ({issue}`8985`)
+* Fix `SHOW ROLES` failure when there are no roles to display. ({issue}`9319`)
+* Fix `EXPLAIN ANALYZE` to show estimates. ({issue}`9396`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:12]: This sound weird .. 

What did it show before? Maybe reword to 

Show estimated values for EXPLAIN ANALYZE queries

File: docs/src/main/sphinx/release/release-363.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+* Fix incorrect result for {func}`round` with precision set to 0. ({issue}`9371`) 
+* Respect `deprecated.legacy-catalog-roles=true` configuration property in `SHOW ROLES`, 
+  `SHOW CURRENT ROLES` and `SHOW ROLE GRANTS` statements. ({issue}`9496`)
+
+## Python client
+
+* Fix column type reported in `cursor.description` for `time with time zone` column. ({issue}`9460`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:19]: Why is the python client release entry here ? Isnt that in a separate repo and separate release cycle? Or is this for any client and just happened to affect the Python client? And it is really an API issue?

File: docs/src/main/sphinx/release/release-363.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+## BigQuery connector
+
+* Fix failure for queries where predicate on `geography`, `array` or `struct` column is pushed down to BigQuery. ({issue}`9391`)
+
+## Cassandra connector
+
+* Add support for Cassandra tuple type. ({issue}`8570`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:27]: code highlighting for tuple maybe?

File: docs/src/main/sphinx/release/release-363.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+## Elasticsearch connector
+
+* Add support for `scaled_float` type. ({issue}`9358`)
+
+## Hive connector
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix data corruption when performing and insert overwrite operation on S3-backed tables. ({issue}`9234`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:36]: performing an insert overwrite operation

File: docs/src/main/sphinx/release/release-363.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+* Fix failure when reading large Parquet files. ({issue}`9469`)
+
+## MySQL connector
+
+* Add support for variable precision `time` type. ({issue}`9339`)
+* Support `CREATE TABLE` and `CREATE TABLE AS` statement for `time` type. ({issue}`9339`)
+
+## Phoenix connector
+
+* Allowing forcing the mapping of certain types to `VARCHAR`. This can be enabled by
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-363.md:56]: This should be in the docs .. not here.

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-363.md (hunk 12 above)
Comment (findepi) [reply docs/src/main/sphinx/release/release-363.md:36]: What kind of corruption is this? @losipiuk

**Review by findepi — APPROVED** [review]

File: docs/src/main/sphinx/release/release-363.md (hunk 12 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-363.md:36]: According to @aczajkowski , it leaves the old data in place and just adds the new data.

File: docs/src/main/sphinx/release/release-363.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix support for comments when adding a new column. Previously, they were silently ignored. ({issue}`9123`)
+* Fix page and block sizing when writing Parquet data. ({issue}`9326`)
+* Fix failure when reading large Parquet files. ({issue}`9469`)
+
+## MySQL connector
+
+* Add support for variable precision `time` type. ({issue}`9339`)
+* Support `CREATE TABLE` and `CREATE TABLE AS` statement for `time` type. ({issue}`9339`)
```
[the remark this answers is not in the document]
Comment (phd3) [reply docs/src/main/sphinx/release/release-363.md:52]: @ebyhr should we say something like

```
* Support `CREATE TABLE` and `CREATE TABLE AS` statements with `time` typed columns. ({issue}`9339`)
```

File: docs/src/main/sphinx/release/release-363.md (hunk 12 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-363.md:36]: @aczajkowski, was there a release where the check wasn't present and it resulted in data corruption?

Comment (aczajkowski) [reply docs/src/main/sphinx/release/release-363.md]: @martint This is the commit when check was added:
https://github.com/trinodb/trino/commit/75831d3268e080dcbd256163365ea3ef7a23a383
So seems version **209**

Comment (martint) [reply docs/src/main/sphinx/release/release-363.md]: Ok, so am I understanding correctly that there was never a data corruption issue, then?
